src/executor/query.rs

```diff
@@ -199,15 +199,20 @@ macro_rules! try_getable_mysql {
     };
 }
 
-macro_rules! try_getable_postgres {
+macro_rules! try_getable_date_time {
     ( $type: ty ) => {
         impl TryGetable for $type {
             fn try_get(res: &QueryResult, pre: &str, col: &str) -> Result<Self, TryGetError> {
                 let _column = format!("{}{}", pre, col);
                 match &res.row {
                     #[cfg(feature = "sqlx-mysql")]
-                    QueryResultRow::SqlxMySql(_) => {
-                        panic!("{} unsupported by sqlx-mysql", stringify!($type))
+                    QueryResultRow::SqlxMySql(row) => {
+                        use chrono::{DateTime, Utc};
+                        use sqlx::Row;
+                        row.try_get::<Option<DateTime<Utc>>, _>(_column.as_str())
+                            .map_err(|e| TryGetError::DbErr(crate::sqlx_error_to_query_err(e)))
+                            .and_then(|opt| opt.ok_or(TryGetError::Null))
+                            .map(|v| v.into())
```

Comment:
For MySQL, get it out as `Option<DateTime<Utc>>` then call `v.into()` to convert it into `DateTime<FixedOffset>`.

Using the conertsion method provided by `chrono`
- https://docs.rs/chrono/0.4.19/chrono/struct.DateTime.html#impl-From%3CDateTime%3CUtc%3E%3E